graphistry/Plottable.py

```diff
@@ -75,6 +75,7 @@ class Plottable(object):
     _entity_to_index : dict
     _index_to_entity : dict
 
+    DGL_graph: Optional[Any]
```

Comment:
rename to _dgl_graph ?

Reply:
yes -- I'll do this on current `cleanup: dbscan and infer_graph` branch